zendesk/provider.go

```diff
@@ -43,6 +43,10 @@ func Provider() terraform.ResourceProvider {
 			"zendesk_attachment":   resourceZendeskAttachment(),
 		},
 
+		DataSourcesMap: map[string]*schema.Resource{
+			"zendesk_system_field": dataSourceZendeskSystemField(),
```

Comment:
potentially should just rename this to be ticket field as well. Any thoughts?

Reply:
yeah, system field means unremovable ticket field.
I think that we could regard them as essentially same.